workers/unity/Packages/com.improbable.gdk.core/Utility/ConnectionUtility.cs

```diff
@@ -16,8 +16,8 @@ public static Connection Connect(ConnectionConfig config, string workerId)
                 case LocatorConfig locatorConfig:
                     return LocatorConnectToSpatial(locatorConfig);
             }
-            throw new InvalidConfigurationException($"Invalid connection config was provided: '{config}' Only" +
-                "ReceptionistConfig and LocatorConfig are supported.");
+            throw new InvalidConfigurationException($"Invalid connection config " +
+                $"was provided: '{config}' Only ReceptionistConfig and LocatorConfig are supported.");
```

Comment:
Sorry, I meant there should be a `\n` between the config and `Only Recep...`

Reply:
lol, added